internal/pkg/fieldpropagator/testdata/src/source/test.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package source
+
+type Source struct {
+	data string
+	id   int
+}
```

Comment:
Just for coverage, I'd like to see coverage for:

[x] getter returns concrete copy of concrete field
[ ] getter returns reference to concrete field
[ ] getter returns reference copy of reference field.
[ ] getter returns concrete dereference of reference field.

If the existing diffset doesn't provide coverage for those additional cases, that could be a future PR.

Reply:
Done. I've also added minimal tests for : 
[x] Conditional return
[x] Multiple return values